control-plane/roles/gardener-shoots/templates/shoot.j2

```diff
@@ -0,0 +1,131 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: "{{ _config.name }}"
+  namespace: "{{ _config.namespace }}"
+  annotations:
+{% if 'managed_seed' in _config %}
+      shoot.gardener.cloud/managed-seed-api-server: "{{ _config.managed_seed.api_server | managed_seed_api_server_annotation() }}"
+{% endif %}
+      cluster.metal-stack.io/name: "{{ _config.name }}"
+      cluster.metal-stack.io/project: "{{ _config.project_id }}"
+      cluster.metal-stack.io/tenant: "{{ metal_control_plane_provider_tenant }}"
+      cluster.metal-stack.io/partition: "{{ _config.partition }}"
+spec:
+  seedName: "{{ _config.seed_name }}"
+  secretBindingName: "{{ _config.name }}-provider-secret"
+  cloudProfileName: metal
+  region: "{{ _config.region }}"
+  purpose: "{{ _config.purpose }}"
+{% if 'managed_seed' in _config %}
+  tolerations: {{ _config.managed_seed.tolerations | to_json }}
+{% endif %}
+  addons:
+    kubernetesDashboard:
+      enabled: false
+{% if 'high_availability_control_plane' in _config %}
+  controlPlane:
+    highAvailability:
+      failureTolerance:
+        type: {{ _config.high_availability_control_plane }}
+{% endif %}
+  kubernetes:
```

Comment:
It should be possible to deactivate kube proxy for example Cilium.
```
    kubeProxy:
      enabled: false
```

Reply:
Done.